### PR TITLE
Feature  profile details

### PR DESCRIPTION
**Pull request by rosalsm**

### CODE REVIEW

**Comment by nickycadavillo** [comment]
:+1: :octocat:

**Comment by cmauban** [comment]
Nice start!! :+1: Add the keys from your json file to the skitch!

File: src/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
         <link rel="apple-touch-icon" href="apple-touch-icon.png">
         <!-- Place favicon.ico in the root directory -->
-
+        <link rel="stylesheet" href="octicons/octicons.css">
```
Comment (kellymurray) [inline src/index.html:12]: You can link the CDN here instead of uploading all the font files, etc.

File: src/index.html [hunk 2]
```diff
@@ -17,7 +17,42 @@
     <body>
       <section class="container">
 
-      <section class="small-rect"></section>
+      <section class="small-rect">
+        <section class="content">
+          <section id="pict">
```
Comment (kellymurray) [inline src/index.html:22]: lots of `<sections>` here! Check out other semantic HTML you can use: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Sections_and_Outlines_of_an_HTML5_document

File: src/scss/_custom.scss [hunk 3]
```diff
@@ -1,20 +1,62 @@
...
+.details{
+ list-style: none;
+}
+.data{
+  margin-left: 30px;
+  margin-top: 20px;
+  margin-right: 10px;
+  margin-bottom: 10px;
+}
+.data{
```
Comment (kellymurray) [inline src/scss/_custom.scss:38]: why do you have `.data` two places?

File: src/scss/_custom.scss [hunk 4]
```diff
@@ -1,20 +1,62 @@
...
+  width:200px;
+}
+tr.mumeric > td{
```
Comment (kellymurray) [inline src/scss/_custom.scss:41]: Lots of typos...this might be causing your css to not render correctly.

File: src/scss/_custom.scss [hunk 5]
```diff
@@ -1,20 +1,62 @@
...
   margin-right: 5px;
-  margin-top: 10px;
+}
+.picture{
+  display:block;
+  width: 300px;
+  height: 300px;
 
 }
+.details{
```
Comment (kellymurray) [inline src/scss/_custom.scss:29]: Why do you have `details` several times here?

**Pull request merged**